app/__init__.py

```diff
@@ -2,18 +2,24 @@
 from flask import Flask
 from flask_bcrypt import Bcrypt
 from flask_login import LoginManager
-from app.models import User, Session
+
+from app.models import Session, User
 
 app = Flask(__name__)
 app.config["SECRET_KEY"] = "12342132542@#242@4%891"
+app.config["UPLOADED_PHOTOS_DEST"] = f"app/static/images/users/current_user"
+app.config["ALLOWED_IMAGE_EXTENSIONS"] = [".jpg", ".jpeg", ".png", ".gif"]
 bcrypt = Bcrypt(app)
 login_manager = LoginManager(app)
-cipher_suite = Fernet(Fernet.generate_key())
+
 
 @login_manager.user_loader
 def load_user(user_id):
     session = Session()
     return session.query(User).get(int(user_id))

```

Comment:
create session is unnecessary but query is good because if stay with "return User.get(int(user_id))" it give me this:

AttributeError: type object 'User' has no attribute 'get'